.storybook/stories/config-pages/LocalFooter.stories.tsx

```diff
@@ -25,57 +33,79 @@ type Story = StoryObj<ComponentStoryProps>;
 
 // More on writing stories with args: https://storybook.js.org/docs/react/writing-stories/args
 export const LocalFooterDisplay: Story = {
+  render: ({numberOfLinks, ...args}) => {
+    const linkList = []
+    for (let i=0; i< numberOfLinks; i++) {
+      linkList.push("<a href='http://localhost'>Primary Link</a>")
+    }
+    return <LocalFooter {...args} />
```

Comment:
@pookmish Can you point out where this is wrong? I thought I'd be able to pass the `linkList` into the return like `<LocalFooter {linkList, ...args} />` but that was an error. I want the control to just add more links so the client can see what happens in the display when they add lots of links.

Reply:
you're creating an array, but you're not doing anything with the array. Also links are typically structured objects, so you'd need to do something like `linkList.push({url: "http:/foobar", title: "foo"})`
But then once you build the array, you need to set it to the args. `args.configPage.[your_field_name] = linkList`

Reply:
Do you mean like this? 
```
args: {
    configPage: {
      suLocalFootAction: linkList,
    },
```